public/app/plugins/datasource/loki/querybuilder/operationUtils.ts

```diff
@@ -0,0 +1,270 @@
+import { LabelParamEditor } from '../../prometheus/querybuilder/components/LabelParamEditor';
```

Comment:
I just wanted to mention that I have a personal grudge against the concept of "utils", because it doesn't provide any useful information, and just tends to englobe anything, since any method or function can be considered an "util".

Reply:
Totally noted. Maybe a topic that could be brought up on frontend weekly.